discord_http/enums.py

```diff
@@ -134,6 +134,10 @@ class IntegrationType(BaseEnum):
     guild = 0
     user = 1
 
+class IntegrationExpireBehaviour(BaseEnum):
```

Comment:
This is a duplicate of `enums.ExpireBehaviour`

Reply:
Oh, I didn't see that. My bad.